packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -103,6 +109,35 @@ class MemoryController {
         .toList();
   }
 
+  void initializeLibraryFilters() {}
+
+  Future computeLibraries() async {
```

Comment:
alternatively: we could just use the "Flutter UI isolate". There is code elsewhere that guesses which it is.
There could be a TODO to add a dropdown to switch between isolates if we ever want to really handle the multiple isolate case well.